src/background/messenger.ts

```diff
@@ -19,36 +19,45 @@ export interface ExtensionAdapter {
 }
 
 export default class Messenger {
-    private reporters: Set<(info: ExtensionData) => void>;
     private adapter: ExtensionAdapter;
+    private changeListenerCount: number;
 
     constructor(adapter: ExtensionAdapter) {
-        this.reporters = new Set();
         this.adapter = adapter;
-        chrome.runtime.onConnect.addListener((port) => {
-            if (port.name === 'ui') {
-                port.onMessage.addListener(async (message) => await this.onUIMessage(port, message));
+        this.changeListenerCount = 0;
+        chrome.runtime.onMessage.addListener((message: Message, sender: any, sendResponse: (any) => void) => {
```

Comment:
```suggestion
        chrome.runtime.onMessage.addListener((message: Message, _: any, sendResponse: (any) => void) => {
```

We don't use `sender` as value. < IDE's will go complain about unused values.